Set reference max width and remove old ng-controller ref

rikeoka opened the pull request:
* Limit width of displayed reference to 7REMS and overflow of ellipsis.  Hovering over reference should show full value
* Remove old ng-controler reference


Old Look:
<img width="1090" alt="screen shot 2016-06-23 at 10 42 46 am" src="https://cloud.githubusercontent.com/assets/11447948/16313778/a3b79424-392f-11e6-9390-db3ce255241a.png">

New Look:
<img width="1133" alt="screen shot 2016-06-23 at 11 41 30 am" src="https://cloud.githubusercontent.com/assets/11447948/16315514/7d4e29da-3937-11e6-8cdd-f4ca0f01603a.png">


/cc @zendesk/samson

### Tasks
 - [ ] :+1: from team

### References
 - Jira link:

### Risks
- Level: Low

rikeoka commented:
If desired could make the width configurable so it could be set to like 8, 10, etc without a code change.

grosser commented:
can we truncate via css instead ? ... if it's a sha (40 characters a-f0-9 then cut down to 7 without the ...)

jonmoter commented:
Agreed on truncating in CSS, so we're not removing actual useful data from the page.

rikeoka commented:
Updated to use CSS max-width to 7rems for font size scalability. 

Also removed ng-controller reference that was causing a JS warning to show up in console.

grosser commented:
👍
